AriesFramework/AriesFramework/wallet/Wallet.swift

```diff
@@ -55,10 +55,8 @@ public class Wallet {
                 try await wallet.createWallet(withConfig: walletConfig, credentials: walletCredentials)
                 userDefaults.set(true, forKey: walletExistKey)
             } catch {
-                if let err = error as NSError? {
```

Comment:
I didn't know that `error.localizedDescription` could be used here. Is it print the same message as `err.userInfo["message"]`?

Reply:
I was wrong. eventough I didn't check all the error cases. but, It seems like only when `userInfo` is nil can gives more information instead of `Unknown error`. could you review my updates?